lib/linguist/languages.yml

```diff
@@ -4352,6 +4328,36 @@ Ragel:
   tm_scope: none
   ace_mode: text
   language_id: 317
+Raku:
+  type: programming
+  color: "#0000fb"
+  extensions:
+  - ".6pl"
+  - ".6pm"
+  - ".nqp"
+  - ".p6"
+  - ".p6l"
+  - ".p6m"
+  - ".pl"
+  - ".pl6"
+  - ".pm"
+  - ".pm6"
+  - ".raku"
+  - ".rakumod"
+  - ".rakutest"
```

Comment:
I'm afraid you'll need to remove these extensions for now, as we only support extensions once they've seen [sizeable usage](https://github.com/github/linguist/blob/master/CONTRIBUTING.md#adding-an-extension-to-a-language) on GitHub.com. [Harvester](https://github.com/Alhadis/Harvester) counted only a negligible number of each:

* [`.raku`](https://github.com/search?p=1&q=extension%3Araku+NOT+nothack&type=Code):  71 files, 25 repos, 22 users
* [`.rakumod`](https://github.com/search?q=extension%3Arakumod+NOT+nothack&type=Code): 169 files, 6 repos, 5 users
* [`.rakutest`](https://github.com/search?q=extension%3Arakutest+NOT+nothack&type=Code): 7 files, 3 repos, 3 users

We can always add them later. See #4219.

Reply:
Fair enough. Can those extensions be tracked separately in #4219? Those extensions are new (but are intended to become the main extensions), so I wouldn't expect them to be particularly popular quite yet (that said, they are much more popular than when I checked them).

Reply:
Certainly. I'll ping @pchaigno to update the list, as he's the one responsible for monitoring extension usage over time.